host_core/lib/host_core/web_assembly/imports.ex

```diff
@@ -216,7 +227,11 @@ defmodule HostCore.WebAssembly.Imports do
           end
       end
 
-    %{token | target: target}
+    if target == :unknown do
+      {:error, :alias_not_found, token}
+    else
+      {:ok, Map.put(token, :target, target)}
```

Comment:
They are equivalent but just to keep with the style in this file I'd recommend doing
```elixir
{:ok, %{token | target: target}}
```

Reply:
This `put`s the key in the map and results in the new map